src/libraries/System.Net.Http/src/System/Net/Http/BrowserHttpHandler/BrowserHttpHandler.cs

```diff
@@ -146,6 +192,24 @@ protected internal override async Task<HttpResponseMessage> SendAsync(HttpReques
 
                 requestObject.SetObjectProperty("method", request.Method.Method);
 
+                // Allowing or Disallowing redirects.
+                if (AllowAutoRedirect)
+                {
+                    requestObject.SetObjectProperty("redirect", "follow");
+                }
+                else
+                {
+                    // Here we will set redirect to `manual` so that we do not crash with an exception
+                    //
+                    // https://developer.mozilla.org/en-US/docs/Web/API/Response/type
+                    //
+                    // other issues from whatwg/fetch:
+                    //
+                    // https://github.com/whatwg/fetch/issues/763
+                    // https://github.com/whatwg/fetch/issues/601
+                    requestObject.SetObjectProperty("redirect", "manual");
```

Comment:
Isn't this "breaking" change?

Reply:
Not sure what "breaking" is referring to.  Could you be more specific?

Reply:
I added more information in the comment section of the PR.  Do those answer better the "breaking" change or is more specific than the comments added?

Reply:
Addressed in recent commit to determine if the _allowAutoRedirect was explicitly set or not.